Formula/nauty.rb

```diff
@@ -0,0 +1,63 @@
+class Nauty < Formula
+  desc "Automorphism groups of graphs and digraphs"
+  homepage "http://pallini.di.uniroma1.it"
+  url "http://pallini.di.uniroma1.it/nauty26r12.tar.gz"
+  version "26r12"
+  sha256 "862ae0dc3656db34ede6fafdb0999f7b875b14c7ab4fedbb3da4f28291eb95dc"
+  # doi "10.1016/j.jsc.2013.09.003"
+  # tag "math"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "nauty.a"
+    system "make", "checks" if build.with? "test"
```

Comment:
`build.with? "test"` is negative here, and we don't run checks in general in Homebrew core. Please remove this line.